selfdrive/car/honda/carstate.py

```diff
@@ -219,11 +219,19 @@ def update(self, cp):
                                       cp.vl["DOORS_STATUS"]['DOOR_OPEN_RL'], cp.vl["DOORS_STATUS"]['DOOR_OPEN_RR']])
     self.seatbelt = not cp.vl["SEATBELT_STATUS"]['SEATBELT_DRIVER_LAMP'] and cp.vl["SEATBELT_STATUS"]['SEATBELT_DRIVER_LATCHED']
 
+    # STEER STATUS
     # 2 = temporary; 3 = TBD; 4 = temporary, hit a bump; 5 = (permanent); 6 = temporary; 7 = (permanent)
     # TODO: Use values from DBC to parse this field
     self.steer_error = cp.vl["STEER_STATUS"]['STEER_STATUS'] not in [0, 2, 3, 4, 6]
     self.steer_not_allowed = cp.vl["STEER_STATUS"]['STEER_STATUS'] != 0
-    self.steer_warning = cp.vl["STEER_STATUS"]['STEER_STATUS'] not in [0, 3]   # 3 is low speed lockout, not worth a warning
+    self.steer_warning = cp.vl["STEER_STATUS"]['STEER_STATUS'] not in [0, 3]  # 3 is low speed lockout, not worth a warning
+    
+    # Adds steering warning for low speed lockout on interceptor-enabled vehicles
+    # For safety, it's best we warn the driver that the steering is disabled on cars requiring interceptor for stop and go.
+    # Although for most of the vehicles that steer to zero this is pointless, for some like the 2016 / 2017 Pilot and Ridgeline they only steer to 12mph with Pedal, they need the warnings.
+    if self.CP.enableGasInterceptor and self.CP.carFingerprint in (CAR.PILOT, CAR.ACURA_RDX, CAR.RIDGELINE):
```

Comment:
I would prefer no alerts at low speed. they are repeatable (user gets used to them pretty quickly) and annoying I think. Plus at 12mph steer isn't critical.

Reply:
Alright, sounds good!